locale/nl.lua

```diff
@@ -1,34 +1,149 @@
+                permission = { name = 'permission', help = 'Permission level' },
+            },
+        },
+        removepermission = {
+            help = 'Permissies verwijderen iemand (God Only)',
+            params = {
+                id = { name = 'id', help = 'Speler ID' },
+                permission = { name = 'permission', help = 'Permission level' },
+            },
+        },
+        openserver = { help = 'Open de server voor iedereen (Admin Only)' },
+        closeserver = {
+            help = 'Sluit de server voor mensen zonder permissies (Admin Only)',
+            params = {
+                reason = { name = 'reason', help = 'Reden voor sluiten (optioneel)' },
+            },
+        },
+        car = {
+            help = 'Spawn voertuig (Admin Only)',
+            params = {
+                model = { name = 'model', help = 'Model van het voertuig' },
+            },
+        },
+        dv = { help = 'Voertuig verwijderen (Admin Only)' },
+        givemoney = {
+            help = 'Geef geld aan een speler (Admin Only)',
+            params = {
+                id = { name = 'id', help = 'Speler ID' },
+                moneytype = { name = 'moneytype', help = 'Type geld (contant, bank, crypto)' },
+                amount = { name = 'amount', help = 'Hoeveelheid' },
+            },
+        },
+        setmoney = {
+            help = 'Forceer geld bij een speler (Admin Only)',
+            params = {
+                id = { name = 'id', help = 'Speler ID' },
+                moneytype = { name = 'moneytype', help = 'Type geld (contant, bank, crypto)' },
+                amount = { name = 'amount', help = 'Hoeveelheid' },
+            },
+        },
+        job = { help = 'Bekijk je huidige baan' },
+        setjob = {
+            help = 'Geef een speler een baan (Admin Only)',
+            params = {
+                id = { name = 'id', help = 'Speler ID' },
+                job = { name = 'job', help = 'Naam van de job' },
+                grade = { name = 'grade', help = 'Niveau' },
+            },
+        },
+        gang = { help = 'Bekijk je gang' },
+        setgang = {
+            help = 'Zet een speler in een bepaalde gang (Admin Only)',
+            params = {
+                id = { name = 'id', help = 'Speler ID' },
+                gang = { name = 'gang', help = 'Gang naam' },
+                grade = { name = 'grade', help = 'Gang niveau' },
+            },
+        },
+        ooc = { help = 'OOC chat bericht' },
+        me = {
+            help = 'Laat lokaal een bericht zien, aan spelers in de buurt',
+            params = {
+                message = { name = 'message', help = 'Bericht om te versturen' }
+            },
+        },
+    },
 }
 
-if GetConvar('qb_locale', 'en') == 'nl' then
-    Lang = Locale:new({
-        phrases = Translations,
-        warnOnMissing = true,
-        fallbackLang = Lang,
-    })
-end
+Lang = Lang or Locale:new({
```

Comment:
We need the convar here to properly handle different locales. See https://github.com/Qbox-project/qbx_weed/pull/19

Reply:
Fixed